universal-application-tool-0.0.1/app/models/Applicant.java

```diff
@@ -14,12 +17,15 @@
 import services.applicant.ApplicantData;
 
 /** The ebean mapped class that represents an individual applicant */
+@History
 @Entity
 @Table(name = "applicants")
 public class Applicant extends BaseModel {
   private static final long serialVersionUID = 1L;
   private ApplicantData applicantData;
 
+  @WhenCreated public Instant when_created;
```

Comment:
Nit (blocking): please confirm to java variable naming convention and use `whenCreated`.